roles/ec2-asg/tasks/void-instances.yml

```diff
@@ -15,4 +15,6 @@
 
 - name: Wait for instance termination to settle down
   pause:
-    minutes: 2
\ No newline at end of file
+    minutes: 2
+  when:
+    - not stop_stack|default(False)
```

Comment:
why stop_stack variable here? What does it do?

Reply:
We discussed this when we did the stuff on Wednesday, I didn't plug it in at the time, if we're on a stop stack we don't need to wait for termination, only required on a destroy stack so we can delete the security groups